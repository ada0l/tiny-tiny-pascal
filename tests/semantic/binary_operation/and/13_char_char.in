var a: char; b: char;
begin writeln(a and b); end.