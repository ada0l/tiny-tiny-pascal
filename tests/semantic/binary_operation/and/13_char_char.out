(2, 17): Operator is not overloaded char and char